cmake/Tools/Platform/Android/android_deployment.py

```diff
@@ -437,6 +437,19 @@ def install_apk_to_device(self, target_device):
         self.adb_call(arg_list=['install', '-t', '-r', str(self.apk_path.resolve())],
                       device_id=target_device)
 
+    def path_exists_on_device(self, path, device_id):
+        try:
+            result, output = self.adb_ls(path=path,
```

Comment:
nit: If you don't use the `output` variable, use the `_` placeholder.
`result, _ = self.adb_ls(...`

Reply:
Good call! Done